contracts/governance/Staking/modules/StakingWithdrawModule.sol

```diff
@@ -245,12 +245,9 @@ contract StakingWithdrawModule is IFunctionsList, StakingShared, CheckpointsShar
     ) internal {
         address vesting = vestingConfig.vestingAddress;
 
-        until = _adjustDateForOrigin(until);
+        until = _timestampToLockDate(until);
         _validateWithdrawParams(vesting, amount, until);
 
-        /// @dev Determine the receiver.
-        if (receiver == address(0)) receiver = msg.sender;
```

Comment:
I think it's very unlikely that receiver is zero for this function, but it could be the case, so we need this check.

Reply:
Currently the `_withdrawFromTeamVesting` is internal function which will only be execute by `_cancelTeamVesting` function, and there is a check `require(_receiver != address(0), "receiver address invalid");` that will prevent the receiver being 0 address.
So could we agree here that we don't need this check anymore?

Reply:
Yes, thank you for the clarification.